gatsby-config.js

```diff
@@ -10,7 +13,7 @@ module.exports = {
     {
       resolve: `gatsby-plugin-postcss`,
       options: {
-        postCssPlugins: [require(`postcss-custom-media`)()],
+        postCssPlugins: [postCssCustomMedia(), postCssNested()],
```

Comment:
This works fine, but you can check out `precss` for future times. It's what I use most times. It emulates `SASS` pretty well. You can check out the plugins there.

`autoprefixer` would also work pretty well here.

PostCSS is literally the best thing basically.

There is `cssnano` to minify and then `postcss-purgecss` to remove any CSS you don't use if you import something like Tailwind or Bulma, to remove stuff you are not using. Kinda like tree shaking

Reply:
Like. `precss` actually actually handles nesting through `postcss-nested`, that's why I went with it. I just refrained from using the entire package because if we're not using it there's no reason I guess. :man_shrugging: 

Have to look into the others. Thanks :heart:

Reply:
Perfectly reasonable :+1: